storage/2018-11-09/datalakestore/paths/delete.go

```diff
@@ -44,7 +44,7 @@ func (client Client) Delete(ctx context.Context, accountName string, fileSystemN
 func (client Client) DeletePreparer(ctx context.Context, accountName string, fileSystemName string, path string) (*http.Request, error) {
 	pathParameters := map[string]interface{}{
 		"fileSystemName": autorest.Encode("path", fileSystemName),
```

Comment:
Does the fist parameter to Encode have to be "path" as well?

Reply:
Yes. The first parameter indicates the type of encoding (`path` for the `pathParameters`). There are two path parameters, `fileSystemName` and `path`, as seen in the first parameter here:

```go
autorest.WithPathParameters("/{fileSystemName}/{path}", pathParameters)
```